engine/src/main/java/org/terasology/world/chunks/CoreChunk.java

```diff
@@ -37,56 +37,211 @@
  */
 @API
 public interface CoreChunk {
+    /**
+     * @return Position of the chunk in blocks
```

Comment:
"position of the chunk in blocks"
This is not entirely clear. What do you mean? Do you mean it returns the local position of the chunk within a block, or blocks is the unit (like 3 blocks away from origin) ?